CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Unreleased
 ------
 * [#965](https://github.com/Shopify/shopify-app-cli/pull/965): Remove --no-optional when using npm to create new project
+* [#993](https://github.com/Shopify/shopify-app-cli/pull/993): Add tax_filter Scripts Extension Point
```

Comment:
Since we're a private project type, we haven't been updating the changelog as it doesn't affect any users. Though this may be useful. What do you think? Also I guess I'll ping @Shopify/platform-dev-tools-education for their input 😛

Reply:
Yeah I'm good with not updating it! I just saw the checkbox in the PR template and thought I had to do it 😂

Reply:
We can can clarify the wording of that: https://github.com/Shopify/shopify-app-cli/pull/994